token-locker/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "token-locker"
```

Comment:
```suggestion
name = "rainbow-bridge-near-token-locker"
```

Reply:
Fixed there https://github.com/aurora-is-near/rainbow-token-connector/pull/99/commits/9cd323f709a08bb6ce442d9219e12d4692b04267